pkg/cmd/cli/backup/create.go

```diff
@@ -87,6 +88,7 @@ func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.Var(&o.IncludeResources, "include-resources", "resources to include in the backup, formatted as resource.group, such as storageclasses.storage.k8s.io (use '*' for all resources)")
 	flags.Var(&o.ExcludeResources, "exclude-resources", "resources to exclude from the backup, formatted as resource.group, such as storageclasses.storage.k8s.io")
 	flags.Var(&o.Labels, "labels", "labels to apply to the backup")
+	flags.StringVar(&o.StorageLocation, "storage-location", "default", "location in which to store the backup")
```

Comment:
I'd say let's leave off the default value of "default" - we'll do defaulting of this field in the controller, based on the server flag that'll be implemented in #750

Reply:
Done